### PR TITLE
CSVS-104 Integration tests for secret deleting/non-existing (VaultService) were implemented

### PR DESCRIPTION
**Pull request by MedvedevBW**

### CODE REVIEW

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala [hunk 1]
```diff
@@ -19,5 +19,13 @@
 package com.bwsw.cloudstack.vault.server.vault
 
 object Constants {
-  val tokenNotFoundStatus = 403
+  object Statuses {
+    val tokenNotFoundStatus = 403
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala]: "status" is redundant

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala [hunk 2]
```diff
@@ -61,6 +61,47 @@ class VaultServiceIntegrationTestSuite extends FlatSpec with IntegrationTestsCom
       .header("X-Vault-Token", IntegrationTestsSettings.vaultRootToken)
       .body(jsonTokenId.getBytes("UTF-8")).post()
 
-    assert(responseRevokedToken.getStatus == Constants.tokenNotFoundStatus)
+    assert(responseRevokedToken.getStatus == Constants.Statuses.tokenNotFoundStatus)
+  }
+
+  "VaultService" should "create secret path hierarchy and then delete it" in {
+    val firstSecretPath = "first"
```
Comment (placccebo) [inline src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: rename to secretRootPath

**Review by placccebo — CHANGES_REQUESTED** [review]

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala (hunk 1 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/vault/Constants.scala]: ok

File: src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala (hunk 2 above)
Comment (MedvedevBW) [reply src/it/scala/com/bwsw/cloudstack/vault/server/vault/VaultServiceIntegrationTestSuite.scala]: ok

**Comment by MedvedevBW** [comment]
@placccebo

**Review by placccebo — APPROVED** [review]

**Pull request merged**